api/v01/entities/vrp_result.rb

```diff
@@ -66,6 +73,7 @@ class VrpResultSolutionRoute < Grape::Entity
       expose :end_time, documentation: { type: Integer, desc: 'Give the actual end time of the current route if provided by the solver' }
       expose :geometry, documentation: { type: String, desc: 'Contains the geometry of the route, if asked in first place' }
       expose :initial_loads, using: VrpResultDetailQuantities, documentation: { is_array: true, desc: 'Give the actual initial loads of the route' }
+      expose :costs, using: VRPResultSolutionRouteCosts, documentation: { desc: 'The impact of the current route within the solution cost' }
```

Comment:
Do we want this cost for each activity or each route?

The costs does not make much sense for each activity (for example distance cost will be less for services clump together but it will be high for the first and the last services (depart/arrive depot) -- which affects the time cost as well). Again fixed cost is a cost that is shared by each activity on the route.